code/modules/clothing/clothing.dm

```diff
@@ -402,13 +528,22 @@ BLIND     // can't see anything
 			return 1
 	return 0
 
-
 /obj/item/clothing/obj_destruction(damage_flag)
-	if(damage_flag == "bomb" || damage_flag == "melee")
+	if(damage_flag == "bomb")
 		var/turf/T = get_turf(src)
 		//so the shred survives potential turf change from the explosion.
 		addtimer(CALLBACK_NEW(/obj/effect/decal/cleanable/shreds, list(T, name)), 1)
 		deconstruct(FALSE)
+	else if(!(damage_flag in list("acid", "fire")))

```

Comment:
This has made moths incapable of fully consuming clothing items.

Reply:
![image](https://user-images.githubusercontent.com/51932756/131200878-a2a77041-dae0-4857-ab65-07dc524f9beb.png)